### PR TITLE
Remove dynamic social buttons

### PR DESCRIPTION
**Pull request by rnagle**
See #901.

This PR:

- Removes the dynamic Twitter and Facebook buttons/links from the single post/page template near the byline and replaces them with simple links for the respective share service.
- Removes references to `largo_show_twitter_count` since it's no longer relevant.
- Updates tests related to `largo_post_social_links`.

### CODE REVIEW

**Comment by aschweigert** [comment]
does this affect the twitter/fb follow widgets? seems like it might. I'm ok with killing those but some sites are pretty wedded to them, the twitter one, particularly. we'll probably need some kind of plan to deal with that if we do nuke them.

**Comment by rnagle** [comment]
@aschweigert Yeah, it definitely does affect the twitter and fb widgets. Didn't realize that at first. I added the required scripts back.

Kind of a bummer that we need to load these, but still think we're doing a good thing by not requiring the JS to power the dynamic share buttons.

File: inc/enqueue.php [hunk 1]
```diff
@@ -90,12 +90,9 @@ function whichHeader() {
...
  */
 if ( ! function_exists( 'largo_footer_js' ) ) {
-	function largo_footer_js() { ?>
+	function largo_footer_js() {
 
-
-		<?php
-		// Are the widgets that contain facebook social buttons loaded (or are we on single/author?)
-		if( largo_facebook_widget::is_rendered() || largo_follow_widget::is_rendered() || is_single() || is_author() ) : ?>
+		if( largo_facebook_widget::is_rendered() || largo_follow_widget::is_rendered() || is_single() || is_author() ) : ?>
```
Comment (benlk) [inline inc/enqueue.php]: Do we need the checks for `is_single()` and `is_author()` here if the scripts are no longer used as a result of the page being an author or single page?

File: inc/enqueue.php [hunk 2]
```diff
@@ -107,7 +104,14 @@ function largo_footer_js() { ?>
 		  fjs.parentNode.insertBefore(js, fjs);
 		}(document, 'script', 'facebook-jssdk'));</script>
 
-		<?php endif;
+<?php endif;
+
+		if ( largo_twitter_widget::is_rendered() || largo_follow_widget::is_rendered() || is_single() || is_author() ) : ?>
```
Comment (benlk) [inline inc/enqueue.php]: Same question.

**Comment by benlk** [comment]
Tests pass and the buttons work. I have couple questions on the enqueueing made in comments on the diff. 

It's a minor issue, but the Print button in the post-header social media buttons doesn't use the '[pointer](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor)' cursor used for the other links. This is understandable, because it doesn't have an `<a>`, but we should probably make it use the pointer cursor instead of a combination of the default arrow for the icon and the text-selection `I` cursor for the text of the button.

**Pull request merged**